src/components/Icons/Svg/Massa/Certificate.tsx

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
ban-ts-comment in the whole file!? 🤔

Reply:
yes, that seems to be the standard for all svg we have in the ui-kit